### ISSUE TITLE
Web: cannot tab over home menu

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#114199]
Steps to Reproduce:

1. click into activity bar
2. start to "Tab"
3. reach the main menu (`compact`)

=> 🐛 I cannot tab over the menu

//cc @isidorn

**Comment by sbatten** [comment]
@bpasero I tried to reproduce on windows in web and desktop with compact and could not.

**Comment by bpasero** [comment]
My platform is macOS, Google Chrome or Edge.
